core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -104,13 +106,16 @@ private InetSocketAddress getAddress() {
     return null;
   }
 
-  private void pingMetaService(InetSocketAddress address)
-      throws AlluxioStatusException {
+  private void pingMetaService(InetSocketAddress address) throws AlluxioStatusException {
     GrpcChannel channel = GrpcChannelBuilder.newBuilder(address, mConfiguration).build();
     ServiceVersionClientServiceGrpc.ServiceVersionClientServiceBlockingStub versionClient =
         ServiceVersionClientServiceGrpc.newBlockingStub(channel);
-    versionClient.getServiceVersion(GetServiceVersionPRequest.newBuilder()
-        .setServiceType(ServiceType.META_MASTER_CLIENT_SERVICE).build());
+    try {
+      versionClient.getServiceVersion(GetServiceVersionPRequest.newBuilder()
+          .setServiceType(ServiceType.META_MASTER_CLIENT_SERVICE).build());
+    } catch (StatusRuntimeException e) {
+      throw GrpcExceptionUtils.fromGrpcStatusException(e);
```

Comment:
This should be from `AlluxioStatusException.fromStatusRuntimeException`

Reply:
I'll re-open with that fix after https://github.com/Alluxio/alluxio/pull/8412 merges